components/campaign/CampaignTable.js

```diff
@@ -20,37 +24,103 @@ export default function CampaignTable (props) {
       km_coastlines_add_mod: user.km_coastlines_add + user.km_coastlines_mod,
       km_waterways_add_mod: user.km_waterways_add + user.km_waterways_mod
     }))
+
+  const headerDivs = prepareAllHeaders(tableHeaderNames.CAMPAIGN)
+  const formattedNum = ({ cell: { value } }) => formatDecimal(value)
+  const { headers, rows, prepareRow } = useTable({
+    columns: [
```

Comment:
This seems to subvert our initial goal of avoiding hardcoded values in the tables. I would try to consolidate this as much as possible so we can avoid manually creating each table

Reply:
One idea could be to turn the TableHeaders tooltip component into a react-table custom hook, and so we could have the logic for the tooltip happen within the react-table rendering. We could then have a custom Table component that reads the glossary and has certain react-table hooks to generate the columns. 

The problem I had with just generating the column names is that the accessor methods to the data and the glossary ids are different. We can standardize them somewhere but I'm wondering how to do it in a way that's not brittle.